desktop/lib/store-kit/index.js

```diff
@@ -0,0 +1,63 @@
+var $ = require( 'nodobjc' );
```

Comment:
If you put 'use strict' at the top of the file you can make use of some ES6 like `let` and `const`